### PR TITLE
Add rules and modify a few

### PR DESCRIPTION
**Pull request by sneridagh**
I had a long meeting with Thomas this morning, we could clarify and do some fixing and cleanup of inconsistencies in the light-theme. The conclusions are the rules that I'm writing in this PR.

Other decisions:
- We will work with versioned files, every time Thomas engage with a round of changes for each project. Each project on its own Figma project folder, the versioned files with the date (eg. 2023-06-22)
- All modifications that are not in the current, work in progress file, are canonical.
- No drafts allowed in the canonical files.
- Every work in progress file will be an initial copy of the current canonical one.
- If any modification to a canonical should happen, it should be communicated via ticket in gitlab to the team, and marked appropriately both in the ticket (specifying the nature of it) and a mark in the file (somehow, via a legend text box side by side to the change).

Thomas will finish a cleanup and start working on the new mobiles screens in the light theme in a new draft file.

We will do the same tomorrow for BfS.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,12 +92,38 @@ cd project
...
 The wrappers have the classnames `blocks-group-wrapper` and the name of the background color, eg. `grey`, defaulting to `transparent` if no `backgroundColor` property is set in the styling block wrapper in the block.
 
+### Vertical spacing rules
+
+These main rules spec applies to the theme:
+
+- On each change of color, a vertical padding (both `padding-bottom` and `padding-top`) of `80px` defined with the main variable `$color-block-change-vertical-spacing`.
+- The default bottom margin is defined with the main variable `$block-vertical-space` and set by default to `25px`.
+- [grid] Vertical spacing for grids should be `80px` for both top and bottom, even if the previous and next blocks are of the same color.
+- [grid+grid] When two grids happen side by side and are of the same color. It should be equal to the grid gap, so it's set to `@gutterWidth` and currently `1rem`. It has to be adjusted with a bit of negative margin since he have to cancel the current inner padding in grid cells.
```
Comment (davisagli) [inline README.md:118]:
```suggestion
- [grid+grid] When two grids happen side by side and are of the same color. It should be equal to the grid gap, so it's set to `@gutterWidth` and currently `1rem`. It has to be adjusted with a bit of negative margin to cancel the current inner padding in grid cells.
```

File: README.md [hunk 2]
```diff
@@ -92,12 +92,38 @@ cd project
...
+- [grid+grid] Grids columns belonging to the same grid and same color in small mobile viewports. They should be closer to match the other adjacent ones, so they seem to belong to the same grid set.
+- [footer] The footer has a top vertical spacing of `80px`.
+- [teasers] The last teaser, except if the following is a button, do NOT have a line at the bottom.
```
Comment (davisagli) [inline README.md:121]:
```suggestion
- [teasers] The last teaser, except if the following is a button, does NOT have a line at the bottom.
```

File: README.md [hunk 3]
```diff
@@ -92,12 +92,38 @@ cd project
...
+- [listing] The last listing, except if the following is a button, do NOT have a line at the bottom.
```
Comment (davisagli) [inline README.md:122]:
```suggestion
- [listing] The last listing, except if the following is a button, does NOT have a line at the bottom.
```

**Review by davisagli — APPROVED** [review]
@sneridagh So where/how do I find the current canonical design?

**Comment by davisagli** [comment]
Needs changelog

**Comment by sneridagh** [comment]
@davisagli after cleaning/validating, we will "cut" this canonical version. I proposed to add an asterisk or a green circle emoji mark to it. What do you think?

**Pull request merged**